src/window.rs

```diff
@@ -600,11 +600,11 @@ impl Window {
         self.window.maybe_queue_on_main(|w| w.request_redraw())
     }
 
-    /// Notify the windowing system that you're before presenting to the window.
+    /// Notify the windowing system before presenting to the window.
     ///
-    /// You should call this event after you've done drawing operations, but before you submit
+    /// You should call this event after you've completed drawing operations, but before you submit
```

Comment:
completed is a bit weird since you could think that you need to complete them actually, but drawing is async in general and you don't need to wait.

But I don't really care.